docs/development/design-decisions.md

```diff
@@ -5,64 +5,81 @@ This file documents the design choices as well as configuration options.
 ## Stateless
 
 No state storage is needed on `renovate` or the source code repository apart from what you see publicly (branches, Pull Requests).
-It, therefore, doesn't matter if you stop/restart the script and would even still work if you had it running from two different locations, as long as their configuration was the same.
+It, therefore, doesn't matter if you stop/restart the script.
+The script would even still work if you had it running from two different locations, as long as their configuration was the same.
 
 ## Synchronous Operation
 
-The script current processes repositories and branches within them all synchronously.
+The script processes repositories and branches within them synchronously.
+This has the following benefits:
 
 - Greatly reduces the chance of hitting simultaneous API rate limits
 - Simplifies logging
 
 ## Cascading Configuration
 
-Configuration options applied per-package (e.g. with package rules) override those applied per package-type, which override those per-repository, which override those which are global (all repositories).
+We use a cascading configuration.
+This means that specific configuration options always override more general configuration options.
+
+From most specific to least specific:
+
+1. Per-package configuration (e.g. with package rules)
+1. Package-type configuration
+1. Per-repository configuration
+1. Global configuration
 
 ## Automatic discovery of package file locations
 
 The default behaviour is to auto-discover all package file (e.g. `package.json`) locations in a repository and process them all.
 Doing so means that "monorepos" are supported by default.
-This can be overridden by the configuration option `includePaths`, where you list the file paths manually (e.g. limit to just `package.json` in the root of the repository).
+
+This behaviour can be overridden by the configuration option `includePaths`, where you list the file paths manually.
+You could limit Renovate to only update the `package.json` in the root of the repository and ignore `package.json` files in other directories.
 
 ## Separate Branches per dependency
 
-By default, `renovate` will maintain separate branches per-dependency.
+By default, `renovate` will maintain separate branches for each dependency.
 So if 20 dependencies need updating, there will be at least 20 branches/PRs.
-Although this may seem undesirable, it was considered even less desirable if all 20 were in the same Pull Request and it's up to the users to determine which dependency upgrade(s) caused the build to fail.
+Although this may seem undesirable, it's even less desirable if all 20 were in the same Pull Request.
+That way it would be up to the users to determine which dependency upgrade(s) caused the build to fail.
```

Comment:
I've tried to improve this sentence by splitting it up, but it still a bit of an passive voice construct.
Do you have ideas to get the same point accross in a more active style of voice?